CARTOGRAPHY.md

```diff
@@ -37,3 +37,67 @@ account.
+
+Within the different fields of color use (like areas, lines, symbols and labels) stronger and weaker colors can be used to indicate fairly distinct and meaningful features as opposed to more general and less distinct features.  For areas also consider that small areas often work better with stronger colors while large areas tend to call for weaker tones.
+
+Area outlines should - when used - be in colors that are not too strong and harmonic with the fill color.  Care should be taken that area outlines are not mistaken for line features.
+
+The goal of the map being readable based on general map reading experience dictates adhering to general conventions of color use in maps to some extent.  Such conventions also often represent sensible choices regarding color physiology in general.  The following principles should generally be followed:
+
+* Use of blue for area and line colors should be reserved for water related features.  Symbols and labels in blue color can also be used for other purposes but sparsely and in a tone clearly distinct from all water related features
+* Use of green should be reserved for vegetation related features, in particular in case of area and line colors.
+* Use of red, orange, yellow and brown line colors should be reserved to roads and paths.
+
+Of course there is a continuum of colors and it is not quite clear where green and blue end.  The important thing here is that water related blues and vegetation related greens form identifiable units in color space that allow intuitively identifying these groups of colors when reading the map.
+
+It is an important general principle that features differing strongly in meaning and purpose for the map user should be well differentiated in either color or by other means while fairly similar features should differ little or not at all.  Or in other words: features rendered with similar color should actually be similar in purpose and meaning for the map user.
+
+Transparency should be used very carefully with consideration for the color mixing this can result in and the ambiguities possibly caused by it.
+
+It is generally advisable to design color relationships in perceptual color spaces.  Use of internal color functions like `lighten()` and `darken()` should be considered carefully since they are performed in sRGB.
+
+
+### Zoom levels
+
+An important aspect of styles for interactive maps is that they are used for a large number of different map sizes or zoom levels.  This creates special challenges for map design.  A particular problem with this stems from the fact that the Mercator projection is a variable scale projection so the different zoom levels do not represent specific map scales.  This fact should always be considered when making design decisions.  Styling should always work for the whole map and for the whole range of map scales present in the zoom level in question.  This normally requires testing changes in different areas at different latitudes.
```

Comment:
I'd pull this paragraph out and put in something in general about the style needing to work around the world, city density is often relevant, and in the mercator projection density can change with latitude. I'm not sure what section that would go in.